### DISCUSSION TITLE
注释

### DISCUSSION THREAD

**Discussion by zhaoyixiao** [looly/hutool#2166] [General]
![123](https://user-images.githubusercontent.com/24690755/156084637-e8e7c54a-95f6-43b9-987f-fb19dec421ed.png)   
(笑脸)

**Comment by looly** [comment]
哈哈，谢谢指正。

**Comment by wangliang181230** [comment]
为啥不使用
```
//region 注释内容

......

//endregion
```

这个IDEA里，可以收起来的。

**Reply by looly** [reply]
学到了。我之前都不知道。

**Comment by wangliang181230** [comment]
![图片](https://user-images.githubusercontent.com/1527893/156107736-ff989ab6-bedf-4e46-a6ab-698174906071.png)
![图片](https://user-images.githubusercontent.com/1527893/156107746-c4b25c7a-4288-49a8-9d05-6d95a47c6aaf.png)
